src/get-started/modules/ROOT/pages/advanced-configuration/compliance.adoc

```diff
@@ -2,8 +2,15 @@
 
 == Trigger agent run
 
-Now we have applied our directive, we can trigger its application by connecting to
-our node using ssh:
+Now we have applied our directive, we can trigger its application by using two approaches:
+
+* by using "Trigger Agent Run" straight from the webapp:
+
+image::trigger-agent-run.png[Trigger directive application by trigger an agent run]
```

Comment:
It will probably be more understandable with a screenshot showing an actual remote-run output

Reply:
can't agree more! By default: if it has an UI, show it. If it don't, make a graphe :)